pkg/kubelet/winstats/perfcounter_nodestats.go

```diff
@@ -217,15 +217,24 @@ func (p *perfCounterNodeStatsClient) getCPUUsageNanoCores() uint64 {
 }
 
 func getSystemUUID() (string, error) {
-	result, err := exec.Command("wmic", "csproduct", "get", "UUID").Output()
+	// Use PowerShell cmdlet "Get-CimInstance" instead of "wmic csproduct get
+	// uuid" for the following reasons:
+	// 1. The WMI command-line tool (Wmic) is deprecated. Use PowerShell cmdlets
+	// instead.
+	// https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/hh831568(v=ws.11)#wmi-providers
+	// 2. The PowerShell cmdlet "Get-CimObject" was removed from PowerShell 6
+	// and are no longer supported.
+	// https://github.com/MicrosoftDocs/PowerShell-Docs/issues/5156#issuecomment-558253083
+	command := "(Get-CimInstance -Class Win32_ComputerSystemProduct).UUID"
+	args := []string{"-NoProfile", "-NonInteractive", "-ExecutionPolicy", "Restricted", "-Command", command}
+	result, err := exec.Command("powershell.exe", args...).CombinedOutput()
```

Comment:
i don't know if there is a better alternative from security standpoint. The problem with relative path execution on windows is that local folder is preferred to `path` variable. So theoretically one can elevate priviledges by placing fake `powershel.exe` into the the kubelet folder.

Reply:
Valid point. Using powershell directly is done in other places as well: https://github.com/kubernetes/kubernetes/blob/7046e1d6892e6ae682f79a4ff73bab375753e77f/pkg/volume/util/util.go#L655-L661

Looking up the CMI docs it looks like this key comes from "This value comes from the UUID member of the System Information structure in the SMBIOS information."  It looks like there is a win32 api that would enable access to that value instead of using powershell: https://docs.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-getsystemfirmwaretable#examples. 

Found this stackoverflow that shows how to do: https://stackoverflow.com/a/60720308/697126 though it is definitely a bit more involved to grab those values